oxidation/libparsec/crates/core/Cargo.toml

```diff
@@ -21,11 +21,14 @@ libparsec_crypto = { path = "../crypto", default-features = false }
 libparsec_protocol = { path = "../protocol", default-features = false }
 libparsec_types = { path = "../types", default-features = false }
 
+futures = "0.3.5"
```

Comment:
Move this dependenc to `platform_async` and re-expose it from there.

This is needed to ensure we use a consistent set of primitive for async operations (instead of having each crate needing this stuff to wonder between futures, futures_lite, smol etc.)

Reply:
Done